examples/cli/src/test/java/io/unitycatalog/cli/table/CliTableOperations.java

```diff
@@ -26,20 +27,24 @@ public CliTableOperations(ServerConfig config) {
 
     @Override
     public TableInfo createTable(CreateTable createTableRequest) throws IOException {
-
         StringBuilder columns = new StringBuilder();
         for (ColumnInfo column : createTableRequest.getColumns()) {
             columns.append(column.getName()).append(" ").append(column.getTypeName().name()).append(",");
         }
         columns.deleteCharAt(columns.length() - 1);
 
-        String [] args = addServerAndAuthParams(List.of(
-                "table", "create" ,
-                "--full_name", createTableRequest.getCatalogName() + "." + createTableRequest.getSchemaName() + "." + createTableRequest.getName(),
-                "--columns", columns.toString(),
-                "--format", createTableRequest.getDataSourceFormat() != null ? createTableRequest.getDataSourceFormat().name() : null,
-                "--storage_location", createTableRequest.getStorageLocation() != null ? createTableRequest.getStorageLocation() : null
-        ), config);
+        List<String> argsList = new ArrayList<>();
```

Comment:
It this related to this change?

Reply:
This is was an error in the earlier implementation, and would lead to a null pointer exception if data format is not set in the options (we were always setting the option in our previous tests)

Reply:
can this be a separate PR?

Reply:
There are two extra changes besides the main ones :
* Remove the null check for `typeText` while converting ColumnInfo to StructType
* If DataSourceFormat is null in CreateTable request do not pass it as a null CLI Param.

Both are necessary for the current test to pass properly as I only want to pass the minimal information to make the test run.

* typeText is currently not used at all for the conversion.
* DataSourceFormat if not specified, defaults to Delta in CLI implementation and hence not a required CLI param.